dags/data_quality_checks_dag.py

```diff
@@ -0,0 +1,153 @@
+"""
+Imports local files to S3, then to CrateDB and checks several data quality properties
+
+Prerequisites
+-------------
+In CrateDB, set up tables for temporarily and permanently storing incoming data.
+See the file setup/smart_home_data.sql in this repository.
+"""
+import os
+import logging
+
+from airflow import DAG
+from airflow.utils.task_group import TaskGroup
+from airflow.utils.dates import datetime
+
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from airflow.providers.common.sql.operators.sql import SQLColumnCheckOperator, SQLTableCheckOperator
+from airflow.providers.amazon.aws.transfers.local_to_s3 import LocalFilesystemToS3Operator
+from airflow.providers.amazon.aws.operators.s3_copy_object import S3CopyObjectOperator
+from airflow.providers.amazon.aws.operators.s3_delete_objects import S3DeleteObjectsOperator
+
+from include.data_checks import COL_CHECKS, TABLE_CHECKS
+
+
+S3_BUCKET = os.environ.get("S3_BUCKET")
+ACCESS_KEY = os.environ.get("ACCESS_KEY")
+SECRET_KEY = os.environ.get("SECRET_KEY")
```

Comment:
For compatibility with our other DAGs (table export DAG): There we called the environment variables `ACCESS_KEY_ID` and `SECRET_ACCESS_KEY`. That's also the terminology AWS uses.

Reply:
Sure, that one I can add :)

Reply:
This is also fixed in new commit.